src/bun.js/javascript.zig

```diff
@@ -3616,13 +3607,24 @@ pub const VirtualMachine = struct {
                 .data => |data| {
                     IPC.log("Received IPC message from parent", .{});
                     if (this.globalThis) |global| {
+                        if (data.isObject()) {
+                            if (data.get(global, "cmd")) |prop| {
+                                if (prop.isString()) {
+                                    if (prop.toString(global).getZigString(global).hasPrefix("NODE_")) {
```

Comment:
Why are we doing this? We control the API. We don't need to put ourselves in a position where we need to check if a user made observable changes.